tests/Endpoints/ClientTest.php

```diff
@@ -70,25 +70,28 @@ public function testGetAllIndexes()
         $this->assertIsArray($response);
         $this->assertCount(2, $response);
 
-        $uids = array_column($response, 'uid');
+        $uids = array_map(function ($index) {
+            return $index->getUid();
+        }, $response);
 
         $this->assertContains($indexA, $uids);
         $this->assertContains($indexB, $uids);
     }
 
     public function testShowIndex()
     {
-        $index = 'index';
-        $this->client->createIndex(
-            $index,
-            ['primaryKey' => 'objectID'],
+        $uid = 'index';
+        $index = $this->client->createIndex(
+            $uid,
+            ['primaryKey' => 'objectID']
         );
 
-        $response = $this->client->showIndex($index);
+        $response = $this->client->showIndex($uid);
 
+        $this->assertInstanceOf(Indexes::class, $index);
         $this->assertIsArray($response);
         $this->assertSame('objectID', $response['primaryKey']);
-        $this->assertSame($index, $response['uid']);
+        $this->assertSame($uid, $response['uid']);
     }
 
     public function testDeleteIndex()
```

Comment:
Maybe we can typehint each method (even if it's `: void`) ?

Reply:
You mean the test functions?

Reply:
Yes, even if we return `void`, typing it can force us to not use it later 🙂